api/v1alpha1/runner_types.go

```diff
@@ -28,10 +28,44 @@ type RunnerSpec struct {
 	Repository string `json:"repository"`
 
 	// +optional
-	Image string `json:"image"`
+	RunnerContainers []corev1.Container `json:"runnerContainers,omitempty"`
```

Comment:
`RunnerContainers` sounds like it replaces runner-specific containers only. I prefer just `Containers` so that it becomes:

```
	if len(runner.Spec.Containers) != 0 {
		pod.Spec.Containers = runner.Spec.Containers
	}
```

Reply:
done